CMakeLists.txt

```diff
@@ -353,91 +356,112 @@ if (OPENAL)
     #target_link_libraries(editor ${STEAMWORKS_CWRAPPER_LIBRARIES} -lstdc++)
   endif()
 endif()
-target_link_libraries(editor ${OPENGL_LIBRARIES})
-target_link_libraries(editor ${THREADS_LIBRARIES})
-target_link_libraries(editor -lm)
+target_link_libraries(${EDITOR_EXE_NAME} ${OPENGL_LIBRARIES})
+target_link_libraries(${EDITOR_EXE_NAME} ${THREADS_LIBRARIES})
+target_link_libraries(${EDITOR_EXE_NAME} -lm)
 # We need to link to Winsock if we're on Windows
 if( NOT WIN32 AND NOT APPLE)
   #Remember, Windows and Mac aren't using find_package for FMOD and PNG.
-  target_link_libraries(editor ${PNG_LIBRARY})
+  target_link_libraries(${EDITOR_EXE_NAME} ${PNG_LIBRARY})
   if (FMOD_FOUND)
-    target_link_libraries(editor ${FMOD_LIBRARY})
+	  target_link_libraries(${EDITOR_EXE_NAME} ${FMOD_LIBRARY})
   endif()
 endif()
 if(WIN32)
-  target_link_libraries(editor wsock32 ws2_32)
-  target_link_libraries(editor -lfmodex)
+	target_link_libraries(${EDITOR_EXE_NAME} wsock32 ws2_32)
+	target_link_libraries(${EDITOR_EXE_NAME} -lfmodex)
 endif()
 if (APPLE)
   if (FMOD_ENABLED)
-    target_link_libraries(editor -lfmodex) #Finally manually link fmod for mac.
+	  target_link_libraries(${EDITOR_EXE_NAME} -lfmodex) #Finally manually link fmod for mac.
   endif()
 endif()
-target_link_libraries(editor ${EXTRA_LIBS}) #Apple needs this for OpenGL to work.
+target_link_libraries(${EDITOR_EXE_NAME} ${EXTRA_LIBS}) #Apple needs this for OpenGL to work.
 if (OPENAL)
-  target_link_libraries(editor ${OPENAL_LIBRARY})
+	target_link_libraries(${EDITOR_EXE_NAME} ${OPENAL_LIBRARY})
   if(TREMOR_ENABLED)
-    target_link_libraries(editor ${TREMOR_LIBRARY})
+	  target_link_libraries(${EDITOR_EXE_NAME} ${TREMOR_LIBRARY})
   else()
-    target_link_libraries(editor ${VORBISFILE_LIBRARY})
+	  target_link_libraries(${EDITOR_EXE_NAME} ${VORBISFILE_LIBRARY})
   endif()
 endif()
 
-#install(TARGETS editor
-#  RUNTIME DESTINATION bin COMPONENT Runtime
-#)
+if (NOT APPLE AND UNIX)
+	install(TARGETS ${EDITOR_EXE_NAME}
+		RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR} COMPONENT Runtime
+	)
 
-#install(DIRECTORY data DESTINATION share/barony )
+install(DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/lang DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/barony)
+
+	if (EXISTS books)
+		install(DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/books DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/barony)
+		install(DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/images DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/barony)
+		install(DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/items DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/barony)
+		install(DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/maps DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/barony)
+		install(DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/models DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/barony)
+		install(DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/music DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/barony)
+		install(DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/sound DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/barony)
```

Comment:
this can all be one call to `install(DIRECTORY` — https://cmake.org/cmake/help/v3.3/command/install.html#installing-directories